docs/operations_table.py

```diff
@@ -0,0 +1,73 @@
+from nvidia.dali import backend as b
+import nvidia.dali.ops as ops
+import nvidia.dali.fn as fn
+import nvidia.dali.plugin.pytorch
+import sys
+
+# Dictionary with modules that can have registered Ops
+ops_modules = {
+    'nvidia.dali.ops': nvidia.dali.ops,
+    'nvidia.dali.plugin.pytorch': nvidia.dali.plugin.pytorch
+}
+
+def to_fn_name(full_op_name):
+    tokens = full_op_name.split('.')
+    tokens[-1] = fn._to_snake_case(tokens[-1])
+    return '.'.join(tokens)
+
+def name_sort(op_name):
+    _, module, name = ops._process_op_name(op_name)
+    return '.'.join(module + [name.upper()])
+
+def main(out_filename):
+    cpu_ops = ops.cpu_ops()
+    gpu_ops = ops.gpu_ops()
+    mix_ops = ops.mixed_ops()
+    all_ops = cpu_ops.union(gpu_ops).union(mix_ops)
+    longest_module = max(ops_modules.keys(), key = len)
```

Comment:
I see some repetition between this file and legacy_to_fn_table.py. Maybe you can extract common part into separate module, or merge them and just have two functions that generate different outputs.